docs/production-deployment.rst

```diff
@@ -108,56 +91,60 @@ Carefully read through the default ``values.yaml`` file and override any values
 configuration file.
 
 At a minimum consider setting up an ingress by changing ``api.ingress``, change the number of
-workers in ``workers.replicaCount`` (and possibly ``workers.highmemory.replicaCount``), check you're happy with
-the workers' cpu and memory limits in ``workers.resources``, and finally set the credentials:
+workers in ``workers.replicaCount`` (and possibly ``workers.highmemory.replicaCount``), check
+you're happy with the workers' cpu and memory limits in ``workers.resources``, and finally set
+the credentials:
 
 * ``postgresql.postgresqlPassword``
 * ``redis.password`` (and ``redis-ha.redisPassword`` if provisioning redis)
 * ``minio.accessKey`` and ``minio.secretKey``
 
-You may additionally want to check the persistent volume storageClass and sizes.
 
 Installation
 ~~~~~~~~~~~~
 
 To install the whole system execute::
 
     cd deployment
-    helm install entityservice --namespace=es --name="n1entityservice" --values ``my-deployment.yaml``
+    helm install entityservice --name="anonlink" --values ``my-deployment.yaml``
 
-This can take around 10 minutes the first time you deploy to a new cluster.
+This can take several minutes the first time you deploy to a new cluster.
 
 Run integration tests and an end to end test
 --------------------------------------------
 
-Update the server url by editing the yaml file then create a new job on the cluster::
+Update the server url by editing the ``jobs/integration-test-job.yaml`` file then create a
+new job on the cluster::
 
     kubectl create -f jobs/integration-test-job.yaml
 
 
+
 To view the celery monitor:
 ---------------------------
 
-Find the pod that the monitor is running on then forward the port:
+Note the monitor must be enabled at deployment. Find the pod that the celery monitor is
+running on then forward the port. For example::
 
-::
+    $ kubectl get -n default pod --selector=run=celery-monitor -o jsonpath='{.items..metadata.name}'
+    entityservice-monitor-4045544268-s34zl
 
-    kubectl port-forward entityservice-monitor-4045544268-s34zl 8888:8888
+    $kubectl port-forward entityservice-monitor-4045544268-s34zl 8888:8888
```

Comment:
whitespace between `$` and `kubectl`
Oh, but all other commands in this doc haven't got a `$` sign. Be consistent.